docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -23,8 +23,13 @@ NOTE: KQL autocomplete for `.siem-signals-*` indices is available on the
 
 * Use the date and time filter to define a specific time range. By default, this filter is set to search the last 24 hours.
 * View detection alerts generated by a specific rule. To do this, go to *Detect* -> *Rules*, then select a rule name in the All rules table. The *Rules detail page* displays a comprehensive view of the rule's details, and alert details are displayed in the Alerts table beneath the Trend histogram. Alternatively, you can go to *Detect* -> *Alerts* -> *Manage rules*, then click on a rule name in the All rules table to view the alerts associated with the rule.
-* Visualize alerts by specific parameters by using the *Stack by* drop-down in the Trend histogram. For example, if you select `signal.rule.name`, the histogram displays the total alert counts by rule name.
-* Group alerts by specific parameters and view individual alert counts by using the *Stack by* drop-down in the Count table. For example, if you select `signal.rule.name`, the table displays the number of alerts generated by each rule.
+* Group alerts by specific parameters and view individual alert counts by using the *Stack by* drop-down in the *Count* table. For example, if you select `signal.rule.name`, the table displays the number of alerts generated by each rule.
+* Visualize alerts by specific parameters by using the *Stack by* drop-down in the *Trend* histogram. For example, if you select `signal.rule.name`, the histogram displays the total alert counts by rule name.
+* Hover over a value in the data grid to display the inline actions available, such as *Filter In*, *Filter Out*, and *Add to timeline investigation*. Click the expand icon to open a full context menu of options, including *Show top values* and *Copy to Clipboard*. The options available vary depending on the type of data.
```

Comment:
We'll be having more discussions of icon vs. button likely at our next UX doc leads meeting. But I like to think of icon as a representation of something, but while you can click it, I think of "button" more of something in a menu, etc. that you can click. The ability to add inline images in the new docs system will make describing these graphic elements in the menus easier. 

Interesting tidbit from the Microsoft Style Guide about use of "[icon](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/i/icon)". 
```suggestion
* Hover over a value in the data grid to display available inline actions, such as *Filter In*, *Filter Out*, and *Add to timeline investigation*. Click the expand button to open a full context menu of options, including *Show top values* and *Copy to Clipboard*. The options available vary depending on the type of data.
```

Reply:
I agree with that distinction, and looking forward to more discussions and guidelines on word choices like this!